src/app/sidebar/sidebar.component.html

```diff
@@ -34,7 +34,8 @@
                     <span>Investment groups</span>
                 </li>
             </div>
-            <div [tooltip]="!isUserVerified(states.user) || !isWalletLazyReady(states.wallet) ? ToolTipNotVerified : null">
+<!--            Todo Remove hidden when project market is implemented -->
```

Comment:
```suggestion
            <!-- TODO: Remove hidden when project market is implemented -->
```

This way, all TODOs are visible in Intellij (and VSCode?) under the TODO tab on the bottom.